FAIL
Kernel panic in run: PANIC at ../../threads/synch.c:217 in lock_acquire(): assertion `!lock_held_by_current_thread (lock)' failed.
Call stack: 0x800421898b 0x800420ac3d 0x800421e0c9 0x800421c6b7 0x800420717f 0x800421dedd 0x800421d979 0x80042096e6 0x8004209b04 0x800421e072 0x800421dcf3 0x800421da9c 0x40011c 0x4003a7 0x40044b 0x400fb0
Translation of call stack:
0x000000800421898b: debug_panic (lib/kernel/debug.c:32)
0x000000800420ac3d: lock_acquire (threads/synch.c:220)
0x000000800421e0c9: close (userprog/syscall.c:288)
0x000000800421c6b7: process_exit (userprog/process.c:356 (discriminator 3))
0x000000800420717f: thread_exit (threads/thread.c:339)
0x000000800421dedd: remove (userprog/syscall.c:231)
0x000000800421d979: page_fault (userprog/exception.c:159)
0x00000080042096e6: intr_handler (threads/interrupt.c:352)
0x0000008004209b04: intr_entry (threads/intr-stubs.o:?)
0x000000800421e072: open (userprog/syscall.c:275)
0x000000800421dcf3: syscall_handler (userprog/syscall.c:150)
0x000000800421da9c: no_sti (userprog/syscall-entry.o:?)
0x000000000040011c: (unknown)
0x00000000004003a7: (unknown)
0x000000000040044b: (unknown)
0x0000000000400fb0: (unknown)
